### ISSUE TITLE
Request imported from postman v2.1 collection contains redundant query parameters in url

### ISSUE THREAD

**Issue by roxk** [jenius-apps/nightingale-rest-api-client#132]
**Version**
Windows 10 2004
Nightingale 4.1.6.0

**Steps to reproduce the bug**
1. Import the following postman v2.1 collection:
```
{
	"info": {
		"_postman_id": "d3232d7e-a773-4953-8b80-0b5aa3fd79a5",
		"name": "Test Collection",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "Test request",
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "http://localhost:3001?param1=a&param2=b",
					"protocol": "http",
					"host": [
						"localhost"
					],
					"port": "3001",
					"query": [
						{
							"key": "param1",
							"value": "a"
						},
						{
							"key": "param2",
							"value": "b"
						}
					]
				}
			},
			"response": []
		}
	],
	"protocolProfileBehavior": {}
}
```
2. Observe Test request's url field has duplicated query parameters

**Expected behavior**
The url field contains no duplicated query parameters.

**Actual behavior**
The url field has redundant query parameters.

**Screenshot**
Notice in the url field the query parameters part starting from `?` is duplicated
![image](https://user-images.githubusercontent.com/16918354/96876394-fd346f80-14aa-11eb-8f52-e82e6aa1a323.png)

**Comment by dpaulino** [comment]
Oops, I did not mean to close this yet.

**Comment by dpaulino** [comment]
Fixed in 4.3.2
